docs/guides/auth/github.md

```diff
@@ -1,7 +1,5 @@
 # Authentication guide - GitHub
 
-*Note:* This guide was written before the renaming. Just replace `HackMD` with `CodiMD` in your mind :smile: thanks!
```

Comment:
Why is this statement removed and not 'CodiMD' replaced with "HedgeDoc"?

Reply:
Because the word "hackmd" was replaced with "hedgedoc"

Reply:
I don't see any updated screenshots for that file in this PR.

Reply:
done